inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java

```diff
@@ -58,6 +59,37 @@
         return Response.success(clusterInfoService.list(request));
     }
 
+    @RequestMapping(value = "/common/save", method = RequestMethod.GET)
+    @ApiOperation(value = "Add a cluster info")
+    @OperationLog(operation = OperationType.CREATE)
+    public Response<Integer> saveCluster(@RequestBody ClusterInfo clusterInfo) {
+        String currentUser = LoginUserUtil.getLoginUserDetail().getUserName();
+        return Response.success(clusterInfoService.save(clusterInfo, currentUser));
+    }
+
+    @RequestMapping(value = "/common/get/{id}")
+    @ApiOperation(value = "Query cluster information of the common")
+    @ApiImplicitParam(name = "id", value = "common cluster ID", dataTypeClass = Integer.class, required = true)
+    public Response<ClusterInfo> getCluster(@PathVariable Integer id) {
+        return Response.success(clusterInfoService.get(id));
+    }
+
+    @RequestMapping(value = "/common/update", method = RequestMethod.POST)
```

Comment:
ditto

Reply:
I will modify this problem in the next issue and pull request.